bcda/web/middleware/validation.go

```diff
@@ -30,11 +30,11 @@ type requestkey int
 
 const rk requestkey = 0
 
-func NewRequestParametersContext(ctx context.Context, rp RequestParameters) context.Context {
+func NewCtxRequestParams(ctx context.Context, rp RequestParameters) context.Context {
```

Comment:
This function doesnt look like its doing a whole lot for us, do you think its useful?  As far as I can tell the two are equivalent:
`ctx := NewCtxRequestParams(r.Context(), rp)`
`ctx := context.WithValue(ctx, 0, rp)`
The difference is the requestkey?  I didnt understand the comment on it.

Reply:
it just checks the context for that particular key; it's not really doing anything for us at this point. It looks like it's used in about 11 places; shall I update those or do we want to keep this PR the current size?